docs/src/reference/change_log.md

```diff
@@ -5,6 +5,10 @@ We also use this change log to document new features that maintain backward comp
 
 ## New features since last version update
 
+## v17 (TKTK - July 2025)
+
+- TKTK July 2025: **Breaking change** Snakemake version 8 (or above) is now required. Various aspects of the workflow were incompatible with v8 including our support for remote files and have now been updated. The nextstrain runtimes have been correspondingly updated TODO XXX [PR 1180](https://github.com/nextstrain/ncov/pull/1180)
+

```

Comment:
Reminder to update before merge (the updated runtimes should also allow the CI to pass)

Reply:
Also probably good to mention here that the Google Cloud storage remote input support has been dropped with this update.

Reply:
👍 Ive updated the changelog and so we're good to merge this PR as is sometime in the next ~two days (I put "18 July 2025" which is tomorrow in NZ and the day after in US). I'll leave this conversation unresolved as a reminder if we don't get round to it until next week.

Reply:
Thanks @jameshadfield! I'll update to "17 July 2025" after docker-base has released and then merge/release this PR to get every squared away today.